ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py

```diff
@@ -729,3 +729,53 @@ def test_jax_numpy_std(
         atol=1e-3,
         rtol=1e-3,
     )
+
+
+# nanmean
+@handle_frontend_test(
+    fn_tree="jax.numpy.nanmean",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        valid_axis=True,
+        force_int_axis=True,
+        large_abs_safety_factor=2,
+        safety_factor_scale="log",
+        allow_nan=True,
+        allow_inf=True,
+    ),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    keepdims=st.booleans(),
+    where=np_helpers.where(),
+)
+def test_jax_numpy_nanmean(
+    dtype_x_axis,
+    dtype,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    where,
+    keepdims,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    where, input_dtypes, test_flags = np_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtypes,
+        test_flags=test_flags,
+    )
+    np_helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        all_aliases=["numpy.nanmean"],
```

Comment:
This arg here is throwing an error as an unexpected keyword arg

Reply:
Hello James ( @jkeane508 ),
I have removed this line === all_aliases=["numpy.nanmean"], === from the code to see if there would be any difference 
would you please check with me the result of this change
thank you so much